lib/matplotlib/mpl-data/stylelib/_classic_test.mplstyle

```diff
@@ -294,14 +294,13 @@ legend.facecolor     : inherit   # legend background color (when 'inherit' uses
 legend.edgecolor     : inherit   # legend edge color (when 'inherit' uses axes.edgecolor)
 
 
-
 ### FIGURE
 # See http://matplotlib.org/api/figure_api.html#matplotlib.figure.Figure
 figure.titlesize : medium     # size of the figure title
 figure.titleweight : normal   # weight of the figure title
 figure.figsize   : 8, 6    # figure size in inches
 figure.dpi       : 80      # figure dots per inch
-figure.facecolor : 0.75    # figure facecolor; 0.75 is scalar gray
+figure.facecolor : w       # figure facecolor; 0.75 is scalar gray
```

Comment:
"0.75 is scalar gray" is a bit mysterious now...

Reply:
edited out